src/services/__mocks__/store.ts

```diff
@@ -38,7 +38,13 @@ export const createFakeStore = (workspaces: che.Workspace[]): Store => {
       workspaceId: '',
       recentNumber: 5,
     },
-    branding: {} as any,
+    branding: {
+      data: {
+        docs: {
+          devfile: 'https://www.eclipse.org/che/docs/che-7'
```

Comment:
is it just a test value? then I would avoid using here a real value to avoid confusion because on day it will be out dated. But probably I'm missing something

Reply:
It is just a  test value

Reply:
That's correct remark. 
@olexii4 please provide there a stub text for devfile docs.

Reply:
I have fixed.